recipes/gentle/meta.yaml

```diff
@@ -0,0 +1,40 @@
+package:
+  name: gentle
+  version: 1.9.5
+
+source:
+  git_url: https://github.com/GENtle-persons/gentle-m
```

Comment:
can we use a release tarball here?

Reply:
Released current master as 1.9.5 alpha 1 .